src/app/modules/verification/verification.routes.ts

```diff
@@ -1,65 +1,74 @@
 import { celebrate, Joi } from 'celebrate'
-import { Express } from 'express'
+import { Router } from 'express'
 
 import verifiedFieldsFactory from './verification.factory'
 
-const mountVfnRoutes = (app: Express): void => {
-  const formatOfId = Joi.string().length(24).hex().required()
-  app.route('/transaction').post(
-    celebrate({
-      body: Joi.object({
-        formId: formatOfId,
-      }),
+const VfnRouter = Router()
```

Comment:
Can call export here, don't need to export only at the end

```suggestion
export const VfnRouter = Router()
```